src/main/java/edu/greenblitz/tobyDetermined/commands/rotatingBelly/RotateOutOfDoor.java

```diff
@@ -0,0 +1,15 @@
+package edu.greenblitz.tobyDetermined.commands.rotatingBelly;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class RotateOutOfDoor extends ParallelDeadlineGroup {
+
+    public RotateOutOfDoor() {
+        super(
+                new WaitCommand(1),
```

Comment:
Isn't it supposed to be the other order? I think what this command is saying is "wait a second until rotateByPower is finished" and it is supposed to by the other way around

Reply:
nope, first is the deadline, then the commands